stock_request/views/stock_request_order_views.xml

```diff
@@ -0,0 +1,228 @@
+                        <group>
+                            <field
+                                name="warehouse_id"
+                                widget="selection"
+                                groups="stock.group_stock_multi_locations"
+                            />
+                            <field
+                                name="location_id"
+                                groups="stock.group_stock_multi_locations"
+                            />
+                            <field
+                                name="allow_virtual_location"
+                                attrs="{'invisible': True}"
+                            />
+                            <field
+                                name="procurement_group_id"
+                                groups="stock.group_adv_location"
+                            />
+                            <field
+                                name="company_id"
+                                groups="base.group_multi_company"
+                                options="{'no_create': True}"
+                            />
+                        </group>
+                    </group>
+                    <notebook>
+                        <page name="items" string="Items">
+                            <field
+                                name="stock_request_ids"
+                                context="{
+                            'default_expected_date':expected_date,
+                            'default_picking_policy': picking_policy,
+                            'default_warehouse_id': warehouse_id,
+                            'default_location_id': location_id,
+                            'default_procurement_group_id': procurement_group_id,
+                            'default_company_id': company_id,
+                            'default_state': state,
+                            }"
+                                attrs="{'readonly': [('state', '!=', 'draft')]}"
+                            >
+                                <tree editable="bottom">
+                                    <field name="name" attrs="{'readonly': True}" />
+                                    <field name="product_id" />
+                                    <field
+                                        name="product_uom_id"
+                                        options="{'no_open': True, 'no_create': True}"
+                                        groups="uom.group_uom"
+                                    />
+                                    <field
+                                        name="route_id"
+                                        options="{'no_create': True}"
+                                        groups="stock.group_stock_multi_locations"
+                                    />
+                                    <field
+                                        name="route_ids"
+                                        attrs="{'invisible': True}"
```

Comment:
This `attrs` seem not working. May be you need to set `invisible="1"` for columns link this.